sunshine-core/src/test/java/io/github/tatools/sunshine/core/FileSystemPathBaseTest.java

```diff
@@ -36,7 +36,7 @@ public void pathWithFolder() {
     public void exist() {
         MatcherAssert.assertThat(
                 "File is absent",
-                new FileSystemPathBase("src/test/resources/test.properties").exist()
+                new FileSystemPathBase("src/main/java/io/github/tatools/sunshine/core/FileSystemPathBase.java").exist()
```

Comment:
I might be better to store relative path to the separate variable or property (system/config)

Reply:
@IVKlymenko This is relevant only for one test. Do I really need to create a variable for it?

Reply:
@IVKlymenko I'm still waiting for the answer.